src/pages/search/SearchPage.tsx

```diff
@@ -103,59 +103,65 @@ const SearchPage = () => {
-                      history.push({ search: resultParams.toString() });
-                      setValues(emptySearchConfig);
-                    }
-                  }}>
-                  {t('search.result')}
-                </SelectableButton>
-
-                <SelectableButton
-                  data-testid={dataTestId.startPage.personSearchButton}
-                  startIcon={<PersonIcon />}
-                  color="person"
-                  isSelected={personIsSeleced}
-                  onClick={() => {
-                    if (!personIsSeleced) {
-                      const personParams = new URLSearchParams();
-                      personParams.set(SearchParam.Type, SearchTypeValue.Person);
-                      history.push({ search: personParams.toString() });
-                      setValues(emptySearchConfig);
-                    }
-                  }}>
-                  {t('search.persons')}
-                </SelectableButton>
-
-                <SelectableButton
-                  data-testid={dataTestId.startPage.projectSearchButton}
-                  startIcon={<ShowChartIcon />}
-                  color="project"
-                  isSelected={projectIsSelected}
-                  onClick={() => {
-                    if (!projectIsSelected) {
-                      const projectParams = new URLSearchParams();
-                      projectParams.set(SearchParam.Type, SearchTypeValue.Project);
-                      history.push({ search: projectParams.toString() });
-                      setValues(emptySearchConfig);
-                    }
-                  }}>
-                  {t('project.project')}
-                </SelectableButton>
+                <TextField
+                  select
+                  fullWidth
+                  size="small"
+                  value={selectedSearchType}
+                  onChange={(event) => setSelectedSearchType(event.target.value as SearchTypeValue)}
+                  sx={{
+                    '.MuiSelect-select': {
+                      display: 'flex',
+                      gap: '0.5rem',
+                      alignItems: 'center',
+                      bgcolor: personIsSeleced || projectIsSelected ? `${paramsSearchType}.main` : 'registration.main',
+                    },
+                  }}
+                  inputProps={{ 'aria-label': t('common.filter') }}>
+                  <MenuItem
+                    sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center', bgcolor: 'registration.main' }}
+                    value={SearchTypeValue.Result}
+                    onClick={() => {
+                      if (!resultIsSelected) {
+                        const resultParams = new URLSearchParams();
+                        history.push({ search: resultParams.toString() });
+                        setValues(emptySearchConfig);
+                      }
+                    }}>
+                    <NotesIcon fontSize="small" />
```

Comment:
Syting fra min side, du trenger ikke å gjøre noe med det:

Ikonvalgene for prosjekt og resultat vet jeg ikke helt hva jeg synes om.

Spesielt inni denne menyen så ser resultat ut som det skal være en hamburgermeny.

Reply:
Ikonene går igjen på forskjellige sider i applikasjonen, men skal ta det videre :+1:  blir isåfall en egen PR